CHANGELOG.md

```diff
@@ -56,6 +56,45 @@ All notable changes to this project will be documented in this file.
             * `get_ext_cal_last_temp`
             * `get_self_cal_last_date_and_time`
             * `get_self_cal_last_temp`
+        * Measurement library methods. - [#806](https://github.com/ni/nimi-python/issues/806)
+            * `add_waveform_processing`
+            * `clear_waveform_measurement_stats`
```

Comment:
Doesn't this method go with `_fetch_measurement_stats`? Shouldn't that be made public too?

Reply:
_fetch_measurement_stats should remain private as the next PR (#1485) adds a wrapper function that exposes it publicly in a pythonic manner. This PR was to help break up #1485 into smaller pieces and to rectify issues with the functions.py file not accurately representing the output of the internal metadata generation.